src/components/ui/Button/Button.scss

```diff
@@ -0,0 +1,5 @@
+@use '../../../scss/variables' as c;
```

Comment:
Please use alias instead of relative path

Reply:
I realized I should use here var(--c-<color-name>) as it is a variable available from :root (html) so it's no longer relevant